lib/experimental/cmdline/parsers.nim

```diff
@@ -80,7 +81,7 @@ proc parseCli*(T: typedesc[SomeFloat], value: string): T =
   ## Implements `parseCli` for all floats.
   let parsed = parseFloat(value)
 
-  if parsed notin low(T)..high(T):
+  if not (parsed in low(T)..high(T) or parsed.isNaN):
```

Comment:
```suggestion
  if parsed notin low(T)..high(T) and not parsed.isNaN:
```

Use De Morgan's to remove unnecessary nesting.

Though for this case it's better to just remove the range test, since it's always between two `inf`s, making this a no-op.